### ISSUE TITLE
Unhandled exception in event loop. Exception [WinError 995]

### ISSUE THREAD

**Issue by Moaneschien** [prompt-toolkit/python-prompt-toolkit#1311]
when running `Loop The REPL` from the docs after a few data entries:

```
> w
You entered: w

Unhandled exception in event loop:
  File "....\AppData\Local\Programs\Python\Python38\lib\asyncio\proactor_events.py", line 768, in _loop_self_reading
    f.result()  # may raise
  File "....\AppData\Local\Programs\Python\Python38\lib\asyncio\windows_events.py", line 808, in _poll
    value = callback(transferred, key, ov)
  File "....\AppData\Local\Programs\Python\Python38\lib\asyncio\windows_events.py", line 457, in finish_recv
    raise ConnectionResetError(*exc.args)

Exception [WinError 995] The I/O operation has been aborted because of either a thread exit or an application request
Press ENTER to continue...

Hit ctrl-C

> Traceback (most recent call last):
  File "....\Documents\Python\envCurses\cu1.py", line 19, in <module>
    main()
  File "....\Documents\Python\envCurses\cu1.py", line 9, in main
    text = session.prompt('> ')
  File "....\Documents\Python\envCurses\lib\site-packages\prompt_toolkit\shortcuts\prompt.py", line 1013, in prompt
    return self.app.run(set_exception_handler=set_exception_handler)
  File "....\Documents\Python\envCurses\lib\site-packages\prompt_toolkit\application\application.py", line 816, in run
    return loop.run_until_complete(
  File "....\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "....\Documents\Python\envCurses\lib\site-packages\prompt_toolkit\application\application.py", line 627, in run_async
    assert not self._is_running, "Application is already running."
AssertionError: Application is already running.
```

Als crtl-c does not work when in the loop. only after it fails.
Win10, Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)]

**Comment by jonathanslenders** [comment]
Duplicate of https://github.com/prompt-toolkit/python-prompt-toolkit/issues/1023
